packages/features/settings/layouts/SettingsLayoutAppDir.tsx

```diff
@@ -105,7 +105,7 @@ const tabs: VerticalTabItemProps[] = [
   },
   {
     name: "teams",
-    href: "/settings/teams",
+    href: "/teams",
```

Comment:
we already have this in next-config redirects

     {
        source: "/settings/teams",
        destination: "/teams",
        permanent: true,
      },

Reply:
yeah but i think this redirect is causing issues with the layout (dir="") getting screwed up